climpred/prediction.py

```diff
@@ -38,19 +47,62 @@ def compute_perfect_model(
         metric (str): `metric` name, see :py:func:`climpred.utils.get_metric_function`.
         comparison (str): `comparison` name, see
                           :py:func:`climpred.utils.get_comparison_function`.
+        dim (str or list): dimension to apply metric over. default: ['member', 'init']
```

Comment:
I'm still not convinced this is necessary. It makes it complex in my mind, since it can *only* be applied to member or init, right?

Reply:
nope. in the perfect_model framework in can be applied to both dimensions at the same time (what is especially needed for `ACC`). the results for the other deterministic metrics are quite similar when comparing applying to `(dim='init').mean('member')`, `(dim='member').mean('init')` or `(dim=['init','member'])